src/pages/Cart/Cart.js

```diff
@@ -1,7 +1,22 @@
 import React from 'react';
+import CartInfoBox from './CartInfoBox/CartInfoBox';
+import CartDefault from './CartDefault/CartDefault';
+import './Cart.scss';
 
-function Cart() {
-  return <div>Cart1</div>;
+export default function Cart() {
+  return (
+    <main>
```

Comment:
```suggestion
export default function Cart() {
  return (
    <main className="cart">
```

- CRA로는 index.html이 하나인 SPA프로젝트를 구성하는 것이고 이런 상황에서 결과적으로 모든 css파일들은 하나의 html에 한번에 불러와집니다.
- 결과적으로 모든 css파일이 한번에 불러와지기에 이 css파일에서 부여한 스타일이 동일한 className을 가진 다른 컴포넌트의 요소들에게도 영향을 미칠 수 있습니다.
- 따라서 반드시 **최상위 태그의 className을 컴포넌트 이름과 동일하게 부여한 뒤 그 className하위로 네스팅** 적용해주세요
```javascript
const ProductDetail = () => {

	return(
		<section className="productDetail">
			<h1 className="title">Title<h1>
		</section>
	)
}
```

Reply:
적용완료 했습니다!